container/config.yaml

```diff
@@ -1,19 +1,21 @@
----
-dmci:
+dmci: 
   distributors:
     - file
     - pycsw
-  distributor_cache: /workdir
+  distributor_cache: workdir
+  rejected_jobs_path: rejected
```

Comment:
why has this changed?

Reply:
These won't disturb anything. This config file is used while we run dmci locally using docker.
I was not at all able to run dmci locally with previous config settings using docker. 
In k8s  this file is replaced by a valid one.

Reply:
ok, sounds good!